Source/TrackManagerUI.cpp

```
@@ -4023,6 +4415,80 @@ NomadUI::DropResult TrackManagerUI::onDrop(const NomadUI::DragData& data, const
             result.message = "Failed to create source";
         }
         
+        clearDropPreview();
+        clearDropPreview();
+        return result;
+    }
+    
+    // 5. Handle Plugin Drop
+    if (data.type == NomadUI::DragDataType::Plugin) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)